src/documentation/components/Navigation.vue

```diff
@@ -35,6 +35,10 @@
             >
           </li>
           <li><NuxtLink to="/components/phone-input">Phone input</NuxtLink></li>
+          <li><NuxtLink to="/components/price">Price</NuxtLink></li>
```

Comment:
make sure to remove the price component code in case price isn't merged before this one